.github/workflows/collab-guard.yml

```
@@ -5,6 +5,58 @@
+          else
+            echo "✅ PR correctly targets 'develop' branch"
+          fi
+      
+      # Size guard - check for large PRs
+      - name: Check PR size
+        id: pr_size
+        run: |
+          # Get the diff stats
+          DIFF_STATS=$(git diff --stat ${{ github.base_ref }}...HEAD)
+          TOTAL_LINES=$(echo "$DIFF_STATS" | tail -1 | grep -o '[0-9]*' | head -1)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)